src/compiler/file_uploader/index.ts

```diff
@@ -0,0 +1,187 @@
+import { Actor, ActorSubclass, HttpAgent } from '@dfinity/agent';
+import { Secp256k1KeyIdentity } from '@dfinity/identity-secp256k1';
+import { existsSync, createReadStream } from 'fs';
+import { readFile } from 'fs/promises';
+import {
+    getCanisterId,
+    getIdentityName,
+    getWebServerPort
+} from '../../../test';
```

Comment:
Make an issue to move the location of these functions

Reply:
I went ahead and just moved this